apps/api/src/app/admin/admin.service.ts

```diff
@@ -35,6 +37,38 @@ export class AdminService {
     this.baseCurrency = this.configurationService.get('BASE_CURRENCY');
   }
 
+  public async addAssetProfile({
+    dataSource,
+    symbol
+  }: UniqueAsset): Promise<SymbolProfile | never> {
+    try {
+      const assetProfile = await this.dataProviderService.getAssetProfiles([
```

Comment:
`const assetProfiles =`

Reply:
In this case it is always going to be only one assetProfile. So assetProfile makes more sense to me.
But if you want it to be assetProfiles let me know.